application/scripts/install_fontawesome.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Create the folder if it doesn't exist
+mkdir -p app/javascript/fontawesome
+
+# Download ESM versions from JSPM.io
+
+curl -o app/javascript/fontawesome/fontawesome-svg-core.js \
+  https://ga.jspm.io/npm:@fortawesome/fontawesome-svg-core@6.5.0/index.mjs
+
+curl -o app/javascript/fontawesome/free-solid-svg-icons.js \
+  https://ga.jspm.io/npm:@fortawesome/free-solid-svg-icons@6.5.0/index.mjs
+
+curl -o app/javascript/fontawesome/free-regular-svg-icons.js \
+  https://ga.jspm.io/npm:@fortawesome/free-regular-svg-icons@6.5.0/index.mjs
+
+curl -o app/javascript/fontawesome/free-brands-svg-icons.js \
+  https://ga.jspm.io/npm:@fortawesome/free-brands-svg-icons@6.5.0/index.mjs
```

Comment:
shouldn't **fontawesome** in this and the file below be installed by **importmap** in application/config/importmap.rb?

Reply:
I need to remove this one in particular. I installed the CSS version instead.
The CSS version does not require importmap as it is an static CSS.

For the first installation, I used the JS version of Fontawesome with Importmap, but I didn't like the JS setup that I needed to add to update the icons tag and it was downloading the JS from a remote server on the FE.

so in the end, I decided to use the static CSS version.